src/components/RecipeCard/RecipeCard.spec.js

```diff
@@ -75,23 +52,41 @@ describe('RecipeCard', () => {
               path: '/product-details/2',
             },
           ]}
-          showDetails={true}
         />
       </Router>
     )
 
-    const instructionsHeader = screen.getByText('Zubereitung')
+    const button = screen.getByRole('button')
+    expect(button).toBeInTheDocument()
+
+    let instructionsHeader = screen.queryByText('Zubereitung')
```

Comment:
Why `let`?

Reply:
I used let because I am using the same variable again in line 78 to find the element after the button was clicked.